src/components/molecules/BallHolder.tsx

```diff
@@ -0,0 +1,18 @@
+import Ball from "../atom/Ball";
+
+import "./style.css";
+
+const BallHolder = () => {
```

Comment:
What do you think about naming it as `Results`?? It sounds clearer in the domain of the application.